app/app.css

```diff
@@ -34,21 +34,24 @@
   --calcite-shell-panel-min-width: 340px;
 }
 
-calcite-slider {
-  max-width: 90%;
```

Comment:
I had this because the slider was causing the scrollbar to appear vertically. If this isn't the case anymore its fine to remove :)